docarray/typing/tensor/abstract_tensor.py

```diff
@@ -117,3 +117,8 @@ def __docarray_from_native__(cls: Type[T], value: Any) -> T:
     def get_comp_backend() -> Type[AbstractComputationalBackend]:
         """The computational backend compatible with this tensor type."""
         ...
+
+    @staticmethod
+    def __docarray_normalize_value__(val: Any) -> Any:
+        """Normalize a value to be compatible with this tensor type."""
+        return val
```

Comment:
Side note: I know we do this in other places too, but I think the Python convention is that double-dunder `__cool_method__` methods are methods that are called "magically" under the hood, but can be overwritten.
So maybe this should be called just `docarray_normalize_value()`?

Reply:
I refactor the code. I don't have this normalize like this anymore